kubernetes-itests/src/test/java/io/fabric8/kubernetes/PatchIT.java

```diff
@@ -48,12 +51,12 @@
 
   @BeforeClass
   public static void init() {
-    ClusterEntity.apply(CronJobIT.class.getResourceAsStream("/patch-it.yml"));
+    ClusterEntity.apply(PatchIT.class.getResourceAsStream("/patch-it.yml"));
   }
 
   @Before
   public void initNamespace() {
-    this.currentNamespace = session.getNamespace();
+    this.currentNamespace = ClusterEntity.getArquillianNamespace();
```

Comment:
Why is this change required?

Reply:
It may not be.  When I was running the test through my ide I was seeing setup and initNamespace using different namespaces - this made sure they were using the same one.

Reply:
I remember I had added `ClusterEntity.getArquillianNamespace()` to find out active namespace used in tests from static context(where session is not available). I think `session.getNamespace()` should work for your use case.

Reply:
Perhaps because of killing the tests prematurely in the ide or something else, they can return different namespaces - so the setup happens in one namespace, while the session is in another.  Then referencing the session namespace here causes the test to fail.  I'll remove this change.